app/models/participant_profile.rb

```diff
@@ -73,6 +73,20 @@ def approved?
   # delivery_partner
   delegate :delivery_partner, to: :latest_induction_record, allow_nil: true
 
+  def self.eligible_to_change_cohort_and_continue_training(in_cohort_start_year:)
+    billable_states = %w[eligible payable paid].freeze
```

Comment:
not in the scope of this pr but we should have this in declarations as a constant similar to statement line items as outside the scope it's used everywhere and needs cleanup 😂

Reply:
Yeah definitely, it would be nice to tidy all that up